src/RadioGroup/Radio.tsx

```diff
@@ -0,0 +1,91 @@
+import * as classnames from "classnames";
+import * as PropTypes from "prop-types";
+import * as React from "react";
+import * as ReactDOM from "react-dom";
+
+import "./Radio.less";
+
+const propTypes = {
+  checked: PropTypes.bool,
+  children: PropTypes.node.isRequired,
+  className: PropTypes.string,
+  disabled: PropTypes.bool,
+  id: PropTypes.string.isRequired,
+  onSelect: PropTypes.func,
+  tabIndex: PropTypes.number,
+  value: PropTypes.any,
+};
+
+const cssClass = {
+  CHECKED: "Radio--checked",
+  CONTAINER: "Radio",
+  DISABLED: "Radio--disabled",
+  INNER: "Radio--inner",
+  LABEL: "Radio--label",
+  OUTER: "Radio--outer",
+};
+
+const LABEL_ID_PREFIX = "Radio--label";
+let nextLabelIDSuffix = 0;
+const getNextLabelID = () => `${LABEL_ID_PREFIX}--${nextLabelIDSuffix++}`;
+
+/**
+ * Single radio button input.
+ * Usually used in a RadioGroup for providing a list of mutually-exclusive options to the user.
+ */
+export default class Radio extends React.PureComponent {
+  static propTypes = propTypes;
+  static cssClass = cssClass;
+
+  _element = null;
+  _labelID = getNextLabelID();
+
+  focus() {
+    if (this._element) {
+      this._element.focus();
+    }
+  }
+
+  componentDidMount() {
+    this._element = ReactDOM.findDOMNode(this);
```

Comment:
oh yeah, we're still blocked from using React 16-specific features in clever-components directly, since not all dependent repos have been migrated to 16 yet. we'll need to do an update across the repo once we've done that (i think it's just HM remaining 🤞)

Reply:
Ah right totally forgot that components weren't on React 16 yet